docs/sources/mimir/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/deploying-grafana-mimir/getting-started-helm-charts/
+description: Learn how to get started with Grafana Mimir using the Helm chart.
+menuTitle: Getting started using the Helm chart
+title: Getting started with Grafana Mimir using the Helm chart
+weight: 25
+---
+
+Contents have been moved to the [documentation for the Grafana Mimir Helm chart](/docs/helm-charts/mimir-distributed/latest/).
```

Comment:
```suggestion
This documentation has moved to the [Grafana Mimir Helm chart]({{< relref="/docs/helm-charts/mimir-distributed/latest/" >}}).
```

Reply:
I think we need a relref here, oder?

Reply:
Relrefs don't start at an absolute path like `/docs` they would start with `./` or `../` . Also I'm not convinced relref could point to `latest`. I'm changing the wording though.

Reply:
I've updated it , although "for" looks a bit weird, maybe "in" the Grafana Mimir Helm chart documentation ?

Reply:
Yes, please use "in".

Reply:
> Relrefs don't start at an absolute path like `/docs` they would start with `./` or `../` . Also I'm not convinced relref could point to `latest`. I'm changing the wording though.

I tested this against `next` locally.